Changelog.md

```diff
@@ -35,6 +35,7 @@ __New Features__
 - ReportUtilityBills measure:
   - Removes utility rate and PV related arguments in lieu of new utility bill scenarios described inside the HPXML file.
   - Adds detailed calculations for tiered, time-of-use, or real-time pricing electric rates using OpenEI tariff files.
+- Allows modeling on/off thermostat deadband with start-up degredation.
```

Comment:
Still need to call out that this applies only to DX systems -- both in the changelog and the documentation.